bunsen-spark/src/test/java/com/cerner/bunsen/spark/codes/broadcast/BroadcastableValueSetsTest.java

```diff
@@ -224,4 +231,83 @@ public void testGetValueSet() {
     Assert.assertTrue(ImmutableSet.of("EM")
         .containsAll(priorityValues.get("http://hl7.org/fhir/v3/ActPriority")));
   }
+
+  @Test
+  public void testThreadSafety() {
```

Comment:
A comment with a link to the issue might be nice for future reference.

Reply:
Added comment to issue link with https://github.com/cerner/bunsen/pull/84/commits/7dcd5ad9692b0bee88a5fbda0946c5c093a3f66c#diff-25abc7ab03aa00073d54970bac2fb665R235